SignalServiceKit/src/Storage/OWSPrimaryStorage.m

```diff
@@ -326,6 +326,16 @@ - (NSString *)databaseFilePath
     return OWSPrimaryStorage.databaseFilePath;
 }
 
+- (NSString *)databaseFilePath_SHM
```

Comment:
I've made a bunch of changes to the storage classes to support the "backup storage" which we no longer use.  I'm going to leave them since I believe they generally improve things.